client/src/app/management/components/meeting-preview/meeting-preview.component.ts

```diff
@@ -34,7 +34,7 @@ export class MeetingPreviewComponent {
     }
 
     public get userAmount(): number {
-        return this.meeting?.user_ids.length;
+        return this.meeting?.user_ids?.length;
```

Comment:
the gettwe now can return null or undefined. Change return value accordingly. Otherwise return 0 if  meeting or user_ids does not exsit